### PR TITLE
CMCL-1034: pov replacement

### PR DESCRIPTION
**Pull request by glabute**
[Delete any line or section that does not apply]

### Purpose of this PR

CMCL-1034: Replace POV with PanTilt
- Uses InputAxisController to drive input axes
- Has explicitly configurable reference frame
- User manual updated

**Still todo**: 
- Upgrader 
- Deprecate and skeletonize POV

### Testing status

- [ ] Added an automated test
- [x] Passed all automated tests
- [x] Manually tested 

### Documentation status

- [ ] Updated [CHANGELOG](https://keepachangelog.com/en/1.0.0/)
- [ ] Updated README (if applicable)
- [x] Commented all public classes, properties, and methods
- [x] Updated user documentation

### Technical risk

low

### CODE REVIEW

File: com.unity.cinemachine/Runtime/Helpers/InputAxisController.cs [hunk 1]
```diff
@@ -218,7 +209,7 @@ string GetDefaultInputName(int axisIndex)
                 {
                     case 0: return "Mouse X";
                     case 1: return "Mouse Y";
-                    case 2: return ""; // "Mouse ScrollWheel"
+                    case 2: return ""; //"Mouse ScrollWheel";
```
Comment (gaborkb) [inline com.unity.cinemachine/Runtime/Helpers/InputAxisController.cs:212]: Why is this not added by default?

File: com.unity.cinemachine/Runtime/Components/CinemachinePanTilt.cs [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+            if (!IsValid)
+                return;
+            var referenceFrame = GetReferenceFrame();
+            var rot = referenceFrame * Quaternion.Euler(TiltAxis.Value, PanAxis.Value, 0);
+            var up = referenceFrame * Vector3.up;
+            curState.RawOrientation = Quaternion.FromToRotation(curState.ReferenceUp, up) * rot;
+        }
+
+        // Normalize angle value to [-180, 180] degrees.
+        static float NormalizeAngle(float angle) => ((angle + 180) % 360) - 180;
```
Comment (gaborkb) [inline com.unity.cinemachine/Runtime/Components/CinemachinePanTilt.cs:142]: This is not used.

**Review by gaborkb — CHANGES_REQUESTED** [review]
Should we deprecate POV?

OrbitalFollow:
I noticed that the radial axis does not affect the camera's distance. It looks like it does not affect anything.

File: com.unity.cinemachine/Tests/Editor/UpgradeCm2ToCm3Tests.cs [hunk 3]
```diff
@@ -88,7 +93,8 @@ public IEnumerator ConvertAllDefaultOnes(Type type)
             Assert.That(vcamGo.GetComponent<CmCamera>(), Is.Not.Null);
             var newComponent = (CinemachineComponentBase) vcamGo.GetComponent(type);
             Assert.That(newComponent, Is.Not.Null);
-            Assert.That(PublicFieldsEqual(newComponent, componentValues), Is.True);
+            if (newComponent is not CinemachinePanTilt) // TODO: temp fix
```
Comment (glabute) [inline com.unity.cinemachine/Tests/Editor/UpgradeCm2ToCm3Tests.cs:96]: no!  I already added a mechanism for this.  See k_IgnoreComponentsList at top of file

**Pull request merged**